include/oneapi/mkl/blas/detail/cublas/blas_ct.hpp

```diff
@@ -27,706 +28,661 @@
 #include <complex>
 #include <cstdint>
 
-#include "oneapi/mkl/detail/backends.hpp"
-#include "oneapi/mkl/detail/libraries.hpp"
 #include "oneapi/mkl/types.hpp"
-
-#include "onemkl_blas_cublas.hpp"
```

Comment:
This header file has cublas backend functions declaration, I believe we need it to be able to call oneapi::mkl::cublas functions from the wrappers below.

Reply:
added it back